spec/strategy_spec.rb

```diff
@@ -1,10 +1,17 @@
 describe "Authentication using Tiddle strategy", type: :request do
   context "with valid email and token" do
     before do
+      User.destroy_all
+      AuthenticationToken.destroy_all
```

Comment:
Why do we need this? Is it because the tests run on Mongo are not wrapped in a transaction? If so, I'd rather use `database_cleaner` as a dependency than have these explicit calls that clear the database.

Reply:
Sure, database_cleaner sounds like a good way to go.